rclcpp/include/rclcpp/experimental/buffers/ring_buffer_implementation.hpp

```diff
@@ -70,7 +69,7 @@ class RingBufferImplementation : public BufferImplementationBase<BufferT>
 
     write_index_ = next_(write_index_);
     ring_buffer_[write_index_] = std::move(request);
-    TRACEPOINT(ring_buffer_enqueue, static_cast<const void *>(this), write_index_, size_+1, is_full_());
+    TRACEPOINT(ring_buffer_enqueue, static_cast<const void *>(this), write_index_, size_ + 1, is_full_());
```

Comment:
This line is too long and needs to be split:

```suggestion
    TRACEPOINT(
      ring_buffer_enqueue,
      static_cast<const void *>(this),
      write_index_,
      size_ + 1,
      is_full_());
```

Reply:
fixed in [d069cca](https://github.com/ros2/rclcpp/pull/2091/commits/d069ccab0e60ec268c8df3b170321d51883dad1d).